src/token.ts

```
@@ -344,10 +340,11 @@ export const isTooEarly = (ucan: Ucan): boolean => {
 /**
  * JWT algorithm to be used in a JWT header.
  */
-function jwtAlgorithm(keyType: KeyType): string | null {
+function jwtAlgorithm(keyType: KeyType): string {
   switch (keyType) {
+    case "bls12-381": throw new Error("Not implemented yet")
     case "ed25519": return "EdDSA"
     case "rsa": return "RS256"
-    default: return null
+    default: throw new Error("Not a KeyType")
```

Can we instead ``throw new Error(`Unknown KeyType "${keyType}"`)``?